t/t1025-flux-account-projects.t

```diff
@@ -0,0 +1,136 @@
+#!/bin/bash
+
```

Comment:
commit message, why `mv` and not `move` like other messages?  NBD, thought maybe a typo.

Reply:
I used `mv` here to fit the commit title in the (is it 50?) character limit for the title. I can update the titles for these commands so that they are all the same and fit in the character limit. Sorry if that caused any confusion!

Reply:
ahh the 50 char limit, it's fine then ... i just thought maybe you typoed it